.github/workflows/release-process.yml

```diff
@@ -0,0 +1,102 @@
+name: kapp-controller release
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  kapp-controller-release:
+    name: kapp-controller release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v1
+        with:
+          path: src/github.com/${{ github.repository }}
+
+      - name: Install Carvel Tools
+        uses: vmware-tanzu/carvel-setup-action@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          only: ytt, kbld
+          ytt: v0.37.0
+          kbld: v0.31.0
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Run release script
+        run: |
+          set -e -x
+          ./hack/build-release.sh
+
+      - name: Create release draft and upload release yaml
+        uses: softprops/action-gh-release@v1
```

Comment:
assuming we have audited this action, lets lock it down by digest.

Reply:
Added digest for the action: `softprops/action-gh-release@17cd0d34deddf848fc0e7d9be5202c148c270a0a`